docs/generate_docs.py

```diff
@@ -354,12 +350,18 @@ def format_subheader(obj, level=1, in_table=False):
     return call_sig
 
 
-def get_class_methods(obj):
-    members = inspect.getmembers(
-        obj, predicate=lambda x: inspect.isroutine(x) and obj.__name__ in x.__qualname__
-    )
-    public_members = [method for (name, method) in members if not name.startswith("_")]
-    return public_members
+def get_class_methods(obj, methods=None):
+    if methods is None:
+        members = inspect.getmembers(
+            obj,
+            predicate=lambda x: inspect.isroutine(x) and obj.__name__ in x.__qualname__,
+        )
```

Comment:
Can you add a comment explaining what this predicate filters?

Reply:
to be honest, I'm not sure. This code predates me, just moved it into a branch.